lightfm/datasets/movielens.py

```
@@ -4,11 +4,224 @@
+
+
+def identity(n, format="csr", dtype=None):
+    """Create an identity matrix - simple replacement for scipy.sparse.identity"""
+    if format == "csr":
+        mat = CsrMatrix((n, n), dtype=dtype)
+        # Set diagonal elements to 1
+        for i in range(n):
+            mat[i, i] = 1.0
+        return mat
+    else:
+        raise ValueError(f"Format '{format}' not supported")
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)